explorer/lib/explorer_web/live/pages/home/index.ex

```diff
@@ -1,7 +1,23 @@
 defmodule ExplorerWeb.Home.Index do
   require Logger
+  import ExplorerWeb.ChartComponents
   use ExplorerWeb, :live_view
 
+  def get_cost_per_proof_chart_data() do
+    data = Batches.get_fee_per_proofs_of_last_n_batches(100)
+
+    %{
+      data:
+        Enum.map(data, fn {fee_per_proof, _} ->
+          case EthConverter.wei_to_usd(fee_per_proof) do
```

Comment:
also here you are missing the second param in wei_to_usd, which says how many decimals you want. i suggest 2.